airflow/example_dags/tutorial_taskflow_api_etl_virtualenv.py

```diff
@@ -19,24 +19,14 @@
 
 # [START tutorial]
 # [START import_module]
-import json
-
 from airflow.decorators import dag, task
 from airflow.utils.dates import days_ago
 
 # [END import_module]
 
-# [START default_args]
-# These args will get passed on to each operator
-# You can override them on a per-task basis during operator initialization
```

Comment:
Do we have a place we explained about default_args being passed to all tasks? Probably we should have one example

Reply:
@ephraimbuddy We still have an explanation in the `tutorial.py` example DAG.

```python
# [START default_args]
# These args will get passed on to each operator
# You can override them on a per-task basis during operator initialization
default_args = {
    'owner': 'airflow',
    'depends_on_past': False,
    'email': ['airflow@example.com'],
    'email_on_failure': False,
    'email_on_retry': False,
    'retries': 1,
    'retry_delay': timedelta(minutes=5),
    # 'queue': 'bash_queue',
    # 'pool': 'backfill',
    # 'priority_weight': 10,
    # 'end_date': datetime(2016, 1, 1),
    # 'wait_for_downstream': False,
    # 'dag': dag,
    # 'sla': timedelta(hours=2),
    # 'execution_timeout': timedelta(seconds=300),
    # 'on_failure_callback': some_function,
    # 'on_success_callback': some_other_function,
    # 'on_retry_callback': another_function,
    # 'sla_miss_callback': yet_another_function,
    # 'trigger_rule': 'all_success'
}
# [END default_args]
```

When I go through the example DAGs again for Taskflow API updates, I'll see what applicable `default_args` use I can sprinkle in too.

Reply:
Whether or not that is a _good_ example is another question though. There is a lot of blind copy/paste by users in DAG files and maybe it makes sense to update the documentation to list these out in an explicit table rather than hiding in DAG code.

We've been trying to improve the `default_args` use (e.g. not specifying default `default_args` and/or moving useful/applicable `default_args` assignment directly to the `DAG()` instantiation). I could definitely add this table as part of the upcoming PR for example DAGs if you agree on this approach.

Reply:
Makes sense @josh-fell. Thanks for that.